src/coordinator.coffee

```diff
@@ -97,9 +97,10 @@ pingUrl = (address, method, callback) ->
 
 class Coordinator extends EventEmitter
   constructor: (@broker, @options = {}) ->
-    @participants = {}
+    @participants = {} # participantId -> Definition (from discovery)
     @connections = {} # connId -> { queue: opt String, handler: opt function }
     @iips = {} # iipId -> { metadata, data }
+    @nodes = {} # role -> { metadata: {} }
```

Comment:
We should probably also keep `component` here, as that way the nodes+connections would be a full representation of the desired graph